package/src/index.ts

```diff
@@ -0,0 +1,288 @@
+import axios, { type AxiosRequestConfig } from "axios"
+import type { Mode, Region, Season, ValorantMap } from "./types/general.js"
+import type { CrosshairResponse } from "./types/v1-crosshair.js"
+import type { FeaturedItemsResponse } from "./types/v1-featured-items.js"
+import type { ProfileResponse } from "./types/v1-profile.js"
+import type { StoreOffersResponse } from "./types/v1-store-offers.js"
+import type { VersionResponse } from "./types/v1-version.js"
+import type { WebsiteResponse } from "./types/v1-website.js"
+import type { MMRResponse } from "./types/v2-mmr.js"
+import type { MatchesResponse } from "./types/v3-matches.js"
```

Comment:
Why are you importing javascript files here?

Also this files doesn't even exist.

Reply:
See my comment